### DISCUSSION TITLE
Column-level authority control

### DISCUSSION THREAD

**Discussion by yongjinhou** [ByConity/ByConity#488] [Q&A] [answered]
Does ByConity support column-level authority control? If yes, could you provide corresponding pr or design document for learning? I don't seem to have found it. And ByConity appears to provide only a column-level authentication interface.

**Comment by yongjinhou** [comment]
The permission levels of database generally include GLOBAL (authorization to the whole world), DATABASE (authorization to database) and TABLE (authorization to table). I want to know whether ByConity supports COLUMN level permission level and can authorize columns, and whether there is any relevant pr or documentation.

**Reply by Adora627** [reply, marked as answer]
Now ByConity doesn't have column level access control. It will release RBAC as a brief access control this Q3(2023), hopefully in 0.2.0. Combining RBAC and View(SQL statement without data storage), you could make it work as column level access control.
